README.md

```diff
@@ -24,18 +24,3 @@ Once the `eslint-config-nava` package is installed, you can use it by specifying
   }
 }
 ```
-
-### Using the `nava` config with `eslint:recommended`
-
-It's suggested to use the `nava` ruleset alongside the [`eslint:recommended` ruleset](http://eslint.org/docs/rules/).
```

Comment:
We should eventually probably document that our config is based on [Standard](http://standardjs.com/rules.html) so there's a place for people to see examples of passing rules.